common/app/layout/Front.scala

```diff
@@ -259,7 +259,6 @@ case class FaciaContainer(
 
   def isStoryPackage: Boolean = container match {
     case Dynamic(DynamicPackage) => true
-    case Dynamic(DynamicElection) => true // #election2017
     case _ => false
   }
```

Comment:
probably not your code but this pattern match is just `container == Dynamic(DynamicPackage)` now?